Gemfile

```diff
@@ -1,4 +1,7 @@
 source 'https://rubygems.org'
+
+ruby '2.4.2'
```

Comment:
You can use `File.read(".ruby-version").chomp` to avoid this getting out of date.